include/wlr/types/wlr_buffer.h

```diff
@@ -111,6 +112,14 @@ bool wlr_buffer_get_dmabuf(struct wlr_buffer *buffer,
  */
 bool wlr_buffer_get_shm(struct wlr_buffer *buffer,
 	struct wlr_shm_attributes *attribs);
+/**
+ * Creates a wlr_buffer from a wl_resource and lock it. Once the caller is done
```

Comment:
Is "imports" more suitable?

Reply:
Hm, maybe "obtains"?

Reply:
"Transforms"?

Reply:
I'm not a fan of "import" because in the case of DMA-BUFs, this function doesn't import anything. I'm fine with "transform". Other alternatives in the same vein could be "convert" or "turn a wl_resource into a wlr_buffer".

Reply:
I went with transform.